__tests__/app.test.js

```diff
@@ -75,4 +74,61 @@ describe("nc-news", () => {
         });
     });
   });
+  describe("GET /api/article/:article_id", () => {
+    test("responds with status 200", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+    test("responds with an object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article;
+          expect(typeof article === "object").toBe(true);
+          expect(article !== null).toBe(true);
+        });
+    });
+    test("responds with an object with the correct proptypes", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article;
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("body");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+        });
+    });
+    test("responds with the correct object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article;
+          expect(article.author).toBe("butter_bridge");
```

Comment:
You could have combined this and the above test. .toHaveProperty() can take a second value, where you can pass in the value/type assertion that you expect to find at that property.

Reply:
expect(article).toHaveProperty("author", "butter_bridge");

like this? cheers